### PR TITLE
feat(genesys-spark): generate global and reset stylesheets

### PR DESCRIPTION
**Pull request by thomas-c-dillon**
COMUI-2275

Relocates global styles and global style example pages to the genesys-spark package. Generates a global and reset stylesheet as part of building the genesys-spark package. Changes to component style import statements, linting configuration, and example web app bootstrapping were updated to support these changes as well.

Looking for feedback on the current changes. 

Future changes include: documentation around the new stylesheets, adding the reset stylesheet to the example web app (possibly limited to certain pages?), and potentially adding a new example page showing the effect of the global and reset styles on common html elements.

### CODE REVIEW

File: packages/genesys-spark/package.json [hunk 1]
```diff
@@ -10,11 +10,12 @@
     "dist/src/*.d.ts"
   ],
   "scripts": {
-    "build": "rollup -c",
+    "build": "./scripts/build-style-sheets.sh && rollup -c",
```
Comment (daragh-king-genesys) [inline packages/genesys-spark/package.json:13]: Just my preference so ignore if you want.
I would do this as
```
"build": "npm run build.style-sheets && npm run build.rollup",
"build.rollup": "rollup -c",
"build.style-sheets": "./scripts/build-style-sheets.sh",
```

File: packages/genesys-spark-components/src/style/style.scss [hunk 2]
```diff
@@ -2,8 +2,7 @@
  * This file combines all the exported library styles into a single stylesheet for consumption by applications.
  */
 
-@use '~genesys-spark-tokens/dist/css/gse-core.css';
-@use '~genesys-spark-tokens/dist/css/gse-semantic.css';
+@use '~genesys-spark/src/style/global.scss';
```
Comment (daragh-king-genesys) [inline packages/genesys-spark-components/src/style/style.scss:5]: This is why this is not a breaking change right? The stylesheet exported with the components will be unchanged (except maybe for the internal order that is not something we depended on)

**Review by daragh-king-genesys — COMMENTED** [review]
Everything look good to me but I would have expected the need for a genesys-spark readme file update for this as well as we are adding new functionality to to package.

File: packages/genesys-spark-components/src/style/style.scss (hunk 2 above)
Comment (thomas-c-dillon) [reply packages/genesys-spark-components/src/style/style.scss:5]: Correct. In theory, the existing stylesheet should be more or less unchanged. This just allows for people to import the global styles (and the new reset styles) separately - even without using the component.

**Review by daragh-king-genesys — APPROVED** [review]

**Comment by jason-evans-genesys** [comment]
Also looks good to me.

**Review by daragh-king-genesys — APPROVED** [review]

**Pull request merged**